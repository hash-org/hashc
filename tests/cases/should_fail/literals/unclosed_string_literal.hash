// run=fail, stage=parse

"a
