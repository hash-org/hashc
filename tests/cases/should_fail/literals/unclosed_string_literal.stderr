error: unclosed string literal
 --> $DIR/unclosed_string_literal.hash:1:2
1 |   "a
  |    ^^ here
